Spigot-Server-Patches/0002-Paper-config-files.patch

```diff
@@ -123,6 +149,7 @@ index 0000000000000000000000000000000000000000..b8868b86338ce0e89bc74eccccf714b9
 +                doReload(sender);
 +                break;
 +            case "ver":
++                if (!testPermission(sender, "version")) break;
```

Comment:
Wrong location, and is this really necessary? `/version` does its own permissions check.

Reply:
I don't think its in the wrong location because its the only subcommand that has a alias. And I think it probably should have it's own permission, just so that its consistent. It's much more of a pain to exempt one subcommand from a permissions check.

Reply:
wrong location, as it only catches the `ver`, not the `version`

Reply:
grr, no the version check is higher up in that method. with *all* the other checks.

Reply:
resolved by adding a comment for dumbos like me :D